packages/gatsby/src/commands/serve.js

```diff
@@ -36,6 +36,7 @@ module.exports = async program => {
     if (open) {
       console.log(`Opening browser...`)
       openurl(openUrlString)
+        .catch(err => console.log(`warn browser not opened`))
```

Comment:
just for consistency - can this follow same format as one in `develop` command?

Reply:
Sure, I didn't do it because `chalk` was not already imported. Should I go ahead and import it?

Reply:
so let's do it other way around and drop `chalk` in the other one?

Reply:
I could do that, but then it will not be consistent will all other messages in `develop`, which are using `chalk`. 🙂

Reply:
hah, then import it in `serve` too ;) it's no big deal

Reply:
Ok, I also beautified the `serve` output a bit in the process, if that's all right:
![image](https://user-images.githubusercontent.com/2083930/46022135-afda4e00-c0ea-11e8-8c8e-f8f68eb60b21.png)